### PR TITLE
Pokepic

### PR DESCRIPTION
**Pull request by PikalaxALT**
Closes #313

### CODE REVIEW

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: include/pokemon.h
Comment (PikalaxALT) [reply include/pokemon.h]: because it uses NARCs in the pbr directory

File: src/pokepic.c
Comment (PikalaxALT) [reply src/pokepic.c:978]: i'm keeping it there because it denotes what was probably intended

File: src/pokepic.c
Comment (PikalaxALT) [reply src/pokepic.c:1137]: you bitch

File: src/register_hall_of_fame.c
Comment (PikalaxALT) [reply src/register_hall_of_fame.c]: not to match, but yes for visual scoping of these stack objects

Comment (PikalaxALT) [reply src/register_hall_of_fame.c]: same for similar comments below

File: src/battle/battle_02261FD4.c
Comment (adrienntindall) [inline src/battle/battle_02261FD4.c:20]: Use new attr constant here

File: src/battle/battle_02261FD4.c
Comment (adrienntindall) [inline src/battle/battle_02261FD4.c:35]: Use new attr constant here

File: src/register_hall_of_fame.c
Comment (adrienntindall) [inline src/register_hall_of_fame.c:1702]: change to camelCase rather than snake_case

**Review by adrienntindall — DISMISSED** [review]
Couple of very minor things, should take like 2 mins to address and I'll let you merge once you do them so you aren't waiting on my review again

File: asm/include/overlay_41.inc
Comment (red031000) [inline asm/include/overlay_41.inc:41]: Pokepic_Unscan

File: include/pokepic.h
Comment (red031000) [inline include/pokepic.h:264]: IsInactive?

File: lib/include/nitro/gx/gx_sp.h
Comment (red031000) [inline lib/include/nitro/gx/gx_sp.h:6]: I'll go through and clean up the libs but these ifdefs are not necessary

File: src/pokepic.c
Comment (red031000) [inline src/pokepic.c:459]: can this just be animStem
same with whichAnim -> anim

File: src/pokepic.c
Comment (red031000) [inline src/pokepic.c:1029]: TRUE

File: src/pokepic.c
Comment (red031000) [reply src/pokepic.c:1137]: **bruh**

Comment (red031000) [reply src/pokepic.c:1137]: tbf it does look like hell so it's fine

**Review by red031000 — CHANGES_REQUESTED** [review]

File: include/pokepic.h
Comment (PikalaxALT) [reply include/pokepic.h:264]: nah, the comment is wrong

File: asm/include/overlay_41.inc
Comment (PikalaxALT) [reply asm/include/overlay_41.inc:41]: this does not operate on the Pokepic class

**Review by red031000 — APPROVED** [review]

**Pull request merged**
